docs/route-tasks.md

```diff
@@ -0,0 +1,138 @@
+---
+id: route-tasks
+title: How to route Tasks
+sidebar_label: Route Tasks
+---
+
+Consider the specifications of a Worker:
+- A Worker is meant to handle only the Tasks that it is registered to handle. This means that Workers registered to handle Tasks for Workflow A are not capable of handling Tasks for Workflow B, and vice versa. 
+- A Worker runs on a speicific host. This means that Activity Tasks that depend on resources specific to a particular host must be executed by a Worker on that host.
+
+So, for any given Temporal Workflow, how do we make sure a specific Worker is executing a specific Task? 
+
+## Use Task Queues
```

Comment:
Why "Use"? I say drop it.

Reply:
The heading answers the question above it. But I see your point in terms of heading "consistency", i.e. using statements, questions, or topics as headers. I disagree that we should be consistent on that point though as it is all about accurately labeling information sections, and I want to be able to use any of them to best inform a user. Limiting ourselves like that is ... silly IMO, but will concede on this one.